packages/ui/src/components/Visualization/ContextToolbar/ExportDocument/ExportDocument.scss

```diff
@@ -0,0 +1,27 @@
+.export-document-preview-body {
+  > h1 {
+    border-bottom: 1px solid var(--pf-t--global--border--color--100);
+    margin-bottom: 1rem;
+  }
+
+  > h2 {
+    border-bottom: 1px solid var(--pf-t--global--border--color--50);
+    margin-bottom: 0.5rem;
+  }
+}
+
+.export-document-preview-table {
+  border-top: 1px solid var(--pf-t--global--border--color--50);
+
+  --pf-v6-c-table__thead--cell--FontSize: 1rem;
+
+  width: 100%;
+}
+
+.export-document-preview-modal {
+  height: var(--pf-v6-c-modal-box--MaxHeight);
+}
+
+.pf-v6-c-table thead > tr:where(.pf-v6-c-table__tr) > :where(th) {
+  max-width: none;
+}
```

Comment:
This could break when the table structure changes at patternfly's side, maybe we could find another solution

Reply:
Moved to a custom class and placed on `<Th>` in the markdown component mapping :+1: